microsoft-edge/webview2/how-to/set-preview-channel.md

```diff
@@ -0,0 +1,290 @@
+
+Updates of the WebView2 Evergreen Runtime often include new APIs and features.  Some of these updates could potentially break your WebView2 app.  To test the experimental APIs ahead of time and ensure your app's forward-compatibility, you should perform compatibility testing using a preview channel of Microsoft Edge, along with a prerelease version of the WebView2 SDK.
+
+
+## Combinations of the SDK and browser channel 
+
+Experimental WebView2 APIs are for use during development and testing of your WebView2 app, not for a production release of your WebView2 app.
+
+*   During development and testing of your WebView2 app, to use experimental WebView2 APIs, use the following combination:
+    *   A prerelease version of the WebView2 SDK.
+    *   A Microsoft Edge preview channel (Beta, Dev, or Canary).  Canary is recommended.
+
+*   For a production release of your WebView2 app, use the following combination, which doesn't provide experimental WebView2 APIs:
+    *   A release version of the WebView2 SDK.
+    *   The WebView2 Runtime.
+
+To develop a WebView2 application that uses experimental APIs, you need two components:
+*  A _prerelease_ version of the WebView2 SDK.  A release version of the SDK doesn't have experimental WebView2 APIs.
+*  A Microsoft Edge _preview_ channel (Beta, Dev, or Canary), also called a Microsoft Edge _Insiders_ channel.
+   *  The WebView2 Runtime doesn't support the WebView2 experimental APIs.
+   *  The Stable version of Microsoft Edge doesn't support WebView2.
+
+### Relation of the SDK and a channel or Runtime
+
+*   SDK: A rerelease of the WebView2 SDK contains the method signatures for experimental APIs, which allow you to write code using the experimental WebView2 APIs in your app.
+*   Binaries: For your WebView2 code to build and run when using experimental APIs, you need to direct your app to use a Microsoft Edge preview channel (Beta, Dev, or Canary).  The preview channels contain the Microsoft Edge binaries that are needed to render your app when using the experimental APIs.
+
+By default, a WebView2 app automatically tries to find and use the WebView2 Runtime, which doesn't have the experimental WebView2 APIs.  This default allows you to easily develop code with the stable-release WebView2 SDK.  However, when you test a prerelease SDK package (which contains experimental APIs), you need to direct your application to use a specific preview channel of Microsoft Edge.
+<!-- related to "Default channel-search order" section below -->
+
+### Downloading the prerelease SDK and a preview channel
+
+To use experimental APIs, download a prerelease version of the WebView2 SDK from [Microsoft.Web.WebView2 package](https://www.nuget.org/packages/Microsoft.Web.WebView2).
+
+To get a Microsoft Edge preview channel, navigate to [Download Microsoft Edge Insider Channels][MicrosoftedgeinsiderDownload].
+
+
+<!-- intro/overview of 4 approaches -->
+## Approaches to making your app use a specific browser channel
```

Comment:
do all 4 approaches explain how to set back to using the Runtime instead of a preview channel?